static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -496,6 +525,38 @@ export class AccessMethod extends Component {
                     </ProjectPanelSection>
                   )
                 }
+                {
+                  supportsConcatenation && (
+                    <ProjectPanelSection
+                      customHeadingTag="h4"
+                      heading="Combine Data"
+                      intro="Select from available operations to combine the data."
+                      nested
+                    >
+                      <Form.Group controlId="input__concatinate-subsetting" className="mb-0">
+                        <Form.Check
+                          id="input__concatinate-subsetting"
+                          type="checkbox"
+                          label={
+                            (
+                              <div>
+                                <span className="mb-1 d-block">
+                                  Enable Concatenation
+                                </span>
+                                <span className="mb-1 d-block gray-text">
```

Comment:
Other labels in this file are using the `text-muted` class, would that do something similar to what your `gray-text` class is doing?

If not, and you do need this class, `color: #a0a0a0;` is not the right way to define a color, we have sass variables for all the colors we should be using so we don't have hex codes all over the code